File: muunioi/BAEKJOON/sprout/b10807.java
package BAEKJOON.sprout;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class b10807 {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());
        int[] arr = new int[n];
        String str = br.readLine();
        StringTokenizer st = new StringTokenizer(str, " ");
        for(int i=0; i<n; i++){
            arr[i] = Integer.parseInt(st.nextToken());
        }
        int v = Integer.parseInt(br.readLine());
        int count = 0;
        for(int num : arr) {
            if(num == v) count++;
        }
        System.out.println(count);
    }
}

File: jihyeonjjang/BAEKJOON/sprout/b10807.swift
import Foundation

let num = Int(readLine()!)!
let arr = readLine()!.split(separator: " ").map{ Int(String($0))! }
let comparenum = Int(readLine()!)!
var count = 0

for i in 0..<num {
    if arr[i]==comparenum {
        count+=1
    }
}

print(count)
